src/components/modal/Modal.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import {useState} from 'react'
+import axios from 'axios'
+import { Link } from 'react-router-dom'
+
+import { usePlaylist } from '../../context/PlaylistContext'
```

Comment:
Look up Absolute Imports. Makes your code look clean and tidy and easy to import.

Reply:
Will look this up